modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModal.tsx

```diff
@@ -7,7 +7,7 @@
  * any later version.
  *
  * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * ANY WARRANTY; without even te implied warranty of MERCHANTABILITY or FITNESS
```

Comment:
Even though you change the license text on next commit, this shouldn't be here, as if for any reason next commit gets dropped, this doesn't stay in master. Please, remove this change from this commit.